README.md

```diff
@@ -2,6 +2,24 @@
 
 Here is a list of [Python Conferences](http://www.pycon.org) around the World.
 
-Each CSV file is [compatible with Google Calendar](https://support.google.com/calendar/answer/37118?hl=en). If would like to add a conference, please [submit a pull request](https://github.com/python-organizers/conferences/pulls).
+If would like to add a conference, please [submit a pull request](https://github.com/python-organizers/conferences/pulls).
 
 If you are a conference organizer, we can give you direct access to the repository, but it is your responsibility to update your own event! The goal is to make it easier for **speakers, instructors, sponsors** to find your conference.
+
+## File Format
+
+Each CSV file is [compatible with Google Calendar](https://support.google.com/calendar/answer/37118?hl=en) with additional optional fields.
+
+| Field             | Description                                | Format                                                                 | Notes
+|-------------------|--------------------------------------------|------------------------------------------------------------------------|-----------------------------------
+| Subject           | Event name                                 | Text                                                                   | required, used by Google Calendar
+| Start Date        | First day of event                         | YYYY-MM-DD                                                             | required, used by Google Calendar
+| End Date          | Last day of event                          | YYYY-MM-DD                                                             | used by Google Calendar
+| Location          | Location of the event                      | Text                                                                   | used by Google Calendar
+| Country           | Country code                               | [ISO 3166-1 alpha-3](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) |
+| Venue             | Event venue                                | Text                                                                   |
```

Comment:
It wasn't too clear at first the difference of Venue and Location. Perhaps the description should say "Venue Name", and the note can further explain:  "The name of the Venue, e.g.: Oregon Convention Center "